### PR TITLE
Bugfix :: Canvas background made programmable and fix for alpha not working

### LINKED ISSUE #2776: Alpha value in colour picker  for changing canvas doesn't work
### What is the expected behaviour?
User must be able to adjust alpha value

### What is the current behaviour?
Alpha value is always set to 100

### How to reproduce the issue?

1.Click on setting icon in left sidebar
2.Open Background color of canvas
3.Try changing alpha value

### If the bug is confirmed, would you be willing to submit a PR?
Yes

### PR DESCRIPTION
**Pull request by stepinfwd**
#resolves #2776 #2775

### CODE REVIEW

**Review by sherfin94 — CHANGES_REQUESTED** [review]
Few major issues:

- Whatever code that we type using Fx, gets applied immediately and does not get applied when the app is running. For eg, we should be able to set the value of the color from an input widget, and if the user edits the text on the widget while the app is running, background color should change.
- When I press the popout button for the Fx field, the area that I can type on the portal that appears is small, there is something buggy thats going on. Not fully sure if its related to this PR

https://user-images.githubusercontent.com/20333972/163809169-433b1c25-1ec1-40af-abe7-4830f76c1745.mov

**Comment by stepinfwd** [comment]
Link : https://canvas-toool.herokuapp.com/
@sherfin94

File: frontend/src/Editor/LeftSidebar/SidebarGlobalSettings.jsx [hunk 1]
```diff
@@ -23,6 +29,19 @@ export const LeftSidebarGlobalSettings = ({
...
+    setRealState(currentState);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [currentState.components]);
+
+  React.useEffect(() => {
+    console.log('check', realState);
+    backgroundFxQuery &&
+      globalSettingsChanged('canvasBackgroundColor', resolveReferences(backgroundFxQuery, realState));
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [realState]);
```
Comment (sherfin94) [inline frontend/src/Editor/LeftSidebar/SidebarGlobalSettings.jsx:43]: How about this to make the background color change as soon as the value computed by backgroundFxQuery changes?
```suggestion
  }, [JSON.stringify(resolveReferences(backgroundFxQuery, realState))]);
```

**Review by sherfin94 — CHANGES_REQUESTED** [review]

**Comment by sherfin94** [comment]
Its not working on viewer @stepinfwd, please check.

**Comment by stepinfwd** [comment]
issue fixed @sherfin94

**Pull request merged**